Dockerfile-latest

```diff
@@ -1,4 +1,4 @@
-FROM centos:7
+FROM ansible/ansible-runner:latest
```

Comment:
Since they don't have an rpm based container upstream maybe we can install ansible-runner from EPEL instead.

Reply:
I don't get the impression they are keeping the RPM up to date. I feel like the best way to stay up-to-date is with the container image but I can make the change if that's best.

```
bash-4.2# yum -y install python-ansible-runner
bash-4.2# rpm -qa | grep 'ansible-runner'
python2-ansible-runner-1.0.1-1.el7.noarch
```